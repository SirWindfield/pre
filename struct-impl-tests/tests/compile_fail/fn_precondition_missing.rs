#![feature(proc_macro_hygiene)]
#![feature(stmt_expr_attributes)]

include!("../../../tests/fn_precondition_missing.rs");
