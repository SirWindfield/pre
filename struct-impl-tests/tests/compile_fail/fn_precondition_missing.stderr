error[E0063]: missing field `_custom_must_20point_20to_20_6042_60` in initializer of `read_twice`
  --> $DIR/fn_precondition_missing.rs:12:9
   |
12 |         #[assert_precondition(holds(valid_ptr(ptr), reason = "`ptr` comes from a reference"))]
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `_custom_must_20point_20to_20_6042_60`
